src/components/Item/VisualizationItem/__tests__/ItemHeaderButtons.spec.js

```diff
@@ -1,24 +0,0 @@
-import React from 'react';
```

Comment:
Why delete this test file completely? `ItemHeaderButtons` is still used

Reply:
You're right. I think I had deleted it earlier in the process when I removed that file.